04.class/memoHandler.js

```diff
@@ -0,0 +1,27 @@
+import Enquirer from "enquirer";
+
+export default class MemoHandler {
+  async selectMemoData(memoTitlesAndIds, action) {
+    const choices = await memoTitlesAndIds.map((row) => ({
+      title: row.title,
+      value: row,
+    }));
+
+    const question = {
+      type: "select",
+      name: "memoData",
+      message: `Choose a memo you want to ${action} :`,
+      choices: choices,
```

Comment:
名前と値が同じなのであれば省略記法を使いましょう。`choices` はそんな複雑な処理をしているわけではないので変数に入れなくてもよいかなとも思います。変数に入れるか入れないかはおまかせします。

```suggestion
      choices,
```